### DISCUSSION TITLE
Public comment history in a private Gist

### DISCUSSION THREAD

**Discussion by lusrx** [community/community#22761] [New to GitHub]
<p>Inside a private Gist (secret), I got this message after I made changes to a comment that a user made to my Gist.</p>
<p><img src="https://user-images.githubusercontent.com/22495331/181073049-865b064c-9bec-41e2-9a56-bcb4810c9e05.png" alt="private-in-public-gist" data-base62-sha1="wvWutzxhQ9phf8FZhc2noeoSd81" width="405" height="199"></p>
<p>Does this mean that my private Gist will be turned into a public Gist?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>That just means that anyone who can see the comments can also see the edits that have been made to the comments - it doesn’t change anything about the gist itself.</p>

**Comment by lusrx** [comment]
<p>I see. So the gist is still private (secret), so that’s good. But I’m not sure I understand the concern… I mean I don’t expect to keep secrets among the people (friends) I decide to share the gist link with. Is this what it’s about? Keeping secrets among friends (people I share the gist with)?..</p>
<p>Example…</p>
<ol>
<li>I create a private gist.</li>
<li>I share the URL with Alice and Bob.</li>
<li>Alice comes in and posts a comment about my gist.</li>
<li>Bob comes in and edits the comment that Alice made.
<ol>
<li>By mistake, he pastes in his super secret password.</li>
</ol>
</li>
<li>Alice comes back in and she looks at comment history and finds Bob’s password.</li>
<li>Alice runs away and uses the password to make Bob’s life a living hell.</li>
</ol>
<p>If I understood correctly…</p>
<p>So it’s to prevent this and to alert me about this that I got the popup message above?</p>

**Comment by canuckjacq** [comment]
<p>Yup! It’s just to let you know that editing a comment doesn’t make the old comment go away - it’s important to us that people understand what information is available to others when they use GitHub.</p>
